.github/workflows/tests.yml

```diff
@@ -12,6 +12,11 @@ on:
   pull_request:
     branches-ignore:
       - v[0-9]+.[0-9]+.[0-9x]+-doc
+    paths-ignore:
+      # No need to test on documentation changes
+      - 'doc/**'
+      - 'examples/**'
+      - 'tutorials/**'
```

Comment:
Need to add plot_types.  Note if we get #25209 in we can just test on "galleries/**"

Reply:
:+1: We can add more later no problem. I guess an alternative question for the future is whether we really want these at the top-level of the repository versus living under `doc/galleries`, `doc/plot_types` etc... which would then allow just `doc/**` to be ignored.

Reply:
Ahh, I see. I have not been following the docs reorganization. That does seem like a nice change!